content/W2017/W17-event-ctf.md

```diff
@@ -0,0 +1,26 @@
+title: Security and Networking CTF
```

Comment:
we should add "with CrySP" since we're collaborating. (I know I told you to just copy the last one sorry 😛)